axelor-production/src/main/resources/i18n/messages_fr.csv

```diff
@@ -19,8 +19,10 @@
 "App production",,,
 "Applicable",,,
 "Approval","Homologation",,
+"Are you sur you want to merge those manuf orders?"," Etes-vous certain de vouloir fusionner ces ordres ?",,
 "Automatic email was not sent because no default email account and/or no valid email account was found : please create one.","Le mail automatique n'a pas pu être envoyé car le compte SMTP par défaut n'a été trouvé et/ou celui-ci n'est pas valide : veuillez le configurer",,
 "Automatically plan the manuf. orders generated from sale order",,,
+"Automatically planified after fusion","Planifier automatiquement après fusion",,
```

Comment:
"planified" is... somewhat a valid word. But hardly ever used. Probably better to use "planned" instead.